themes/tiddlywiki/vanilla/base.tid

```diff
@@ -938,6 +938,10 @@ canvas.tc-edit-bitmapeditor  {
 	display: block;
 }
 
+.tc-tiddler-body {
+	clear: both;
```

Comment:
A global `.clearfix` is fine with me, but what strange behaviour would you mean?